R/combinat.R

```diff
@@ -0,0 +1,46 @@
+## Code to read in multiple CSVs, add some detail around number of bedrooms,...
+## bind them together for easier manipulation, and create a Long file for visualization
+
+#Installing and loading requisite package for "pivot_longer" later
+install.packages('tidyr')
+require(tidyr)
+
+
+# Read several CSVs as independent lists for future combination
+# Location Reference is to the root git folder 
+one_bedroom_zhvi <- read.csv('./data/zhvi_1br_by_zip.csv', stringsAsFactors = FALSE)
+two_bedroom_zhvi <- read.csv('./data/zhvi_2br_by_zip.csv', stringsAsFactors = FALSE)
+three_bedroom_zhvi <- read.csv('./data/zhvi_3br_by_zip.csv', stringsAsFactors = FALSE)
+four_bedroom_zhvi <- read.csv('./data/zhvi_4br_by_zip.csv', stringsAsFactors = FALSE)
+fiveplus_bedroom_zhvi <- read.csv('./data/zhvi_5plusbr_by_zip.csv', stringsAsFactors = FALSE)
+
+
+# Add variable to each list with number of bedrooms
+one_bedroom_zhvi$size <- 'oneBedroom'
+two_bedroom_zhvi$size <- 'twoBedroom'
+three_bedroom_zhvi$size <- 'threeBedroom'
+four_bedroom_zhvi$size <- 'fourBedroom'
+fiveplus_bedroom_zhvi$size <- 'fiveplusBedroom'
+
+# Combine like Megazord
+combined <- rbind(one_bedroom_zhvi,two_bedroom_zhvi,three_bedroom_zhvi,four_bedroom_zhvi,fiveplus_bedroom_zhvi)
+
+
+
+# Make Long for visualization purposes. Dropping NAs for ease of viewing (and viz purposes)
+newcombo <- pivot_longer(
+  combined,
+    cols = starts_with("X"),
+    names_to = "Week",
+    values_to = "Cost",
+    values_drop_na = TRUE
```

Comment:
Did we want to Impute instead of drop?  This could improperly skew the data if, say, 5Bed houses are missing the most data.

Reply:
That's a great idea. Perhaps grab the prior date's value? 

`where na: insert prior date's value`-ish

Reply:
You could also tell me to f off when I leave a comment like this.

Insert prior or use the average are both totally normal things to do!

Reply:
I think that's a great enhancement request to go into our trello board when it exists. I'll start thinking about improvements.

Reply:
How many NAs are there? just want to generally know the impact of dropping them. 
TBH, I'm cool dropping them and not imputing if it's not a huge subset of data. (I'm personally not a fan of imputing large amounts data and try to avoid it as much as possible)

otherwise, works for me. approved.